bitnami/kafka/3.2/debian-11/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -240,25 +240,63 @@ kafka_validate() {
 
     if is_boolean_yes "$KAFKA_ENABLE_KRAFT"; then
         if [[ -n "$KAFKA_CFG_BROKER_ID" ]]; then
-            warn "KAFKA_CFG_BROKER_ID Must match what is set in KAFKA_CFG_CONTROLLER_QUORUM_VOTERS"
+            IFS=','
```

Comment:
I would suggest to keep the old IFS and restoring it after.
Other part of the code could relay on the default value for IFS and this would interfere with that:
```
old_IFS=$IFS
IFS="something_new"
run_your_code
IFS=${old_IFS}
````

What do you think ?